src/entercommand.js

```diff
@@ -56,7 +56,10 @@ function enterBlock( model, writer, selection, schema ) {
 	}
 
 	if ( isSelectionEmpty ) {
+		// List of text attributes copied to new line/block.
+		const filteredAttr = getAllowedAttributes( writer.model.schema, selection.getAttributes() );
```

Comment:
AFAIR no `Attr` but `Attributes` (maybe not so inside method as in public API).

Maybe go with  `attributesToCopy` and remove the comment? It should be pretty obious then what happens here so the comment would be not needed. (with conjuction of method rename below).